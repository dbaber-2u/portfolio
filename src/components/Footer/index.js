import React from "react";

function Footer() {
    return (
        <section className="my-footer">
            <br /><br />
            <ul className="flex-row my-footer">
                <li>
                    <a href="https://github.com/dbaber-2u">
                    <img src="github.png"
                    alt="GitHub"
                    />
                    </a>
                </li>
                <li>
                    <a href="https://linkedin.com/dbaber-2u">
                        <img src="../../assets/icons/linkedin.jpg"
                        alt="LinkedIn"
                        />
                    </a>
                </li>
                <li>
                    <a href="https://twitter.com/dbaber-2u">
                        <img src="../../assets/icons/twitter.jpg"
                        alt="Twitter"
                        />
                    </a>
                </li>
            </ul>
        </section>
    )
}

export default Footer;