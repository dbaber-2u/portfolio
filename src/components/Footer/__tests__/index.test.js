import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Footer from '..';

afterEach(cleanup);


describe('Footer component', () => {
    // baseline test
    it('renders', () => {
        render(<Footer />);
    });

    // snapshot test
    it('matches snapshot', () => {
        const { asFragment } = render(<Footer />);

        expect(asFragment()).toMatchSnapshot();
    });
})