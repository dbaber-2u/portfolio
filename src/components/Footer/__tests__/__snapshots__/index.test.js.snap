// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component matches snapshot 1`] = `
<DocumentFragment>
  <footer
    class="my-footer"
  >
    <ul
      class="flex-row"
    >
      <li>
        <a
          href="https://github.com/dbaber-2u"
        >
          <img
            alt="GitHub"
            src="github.png"
          />
        </a>
      </li>
      <li>
        <a
          href="https://linkedin.com/dbaber-2u"
        >
          <img
            alt="LinkedIn"
            src="linkedin.png"
          />
        </a>
      </li>
      <li>
        Twitter
      </li>
    </ul>
  </footer>
</DocumentFragment>
`;
